lib/verbalize/build.rb

```diff
@@ -1,12 +1,13 @@
 module Verbalize
   class Build
-    def self.call(required_keywords = [], optional_keywords = [])
-      new(required_keywords, optional_keywords).call
+    def self.call(required_keywords = [], optional_keywords = [], default_keywords = [])
+      new(required_keywords, optional_keywords, default_keywords).call
     end
 
-    def initialize(required_keywords, optional_keywords)
+    def initialize(required_keywords, optional_keywords, default_keywords)
```

Comment:
we added `default_keywords` here to extract the default values into the built definition...